rel/i18n/emqx_schema.hocon

```diff
@@ -998,8 +1007,8 @@ fields_ws_opts_supported_subprotocols.label:
 mqtt_shared_subscription_strategy.desc:
 """Dispatch strategy for shared subscription.
  - `random`: Randomly select a subscriber for dispatch;
- - `round_robin`: Messages from a single publisher are dispatched to subscribers in turn;
- - `round_robin_per_group`: All messages are dispatched to subscribers in turn;
+ - `round_robin`: Clients in a shared subscription group will consume messages in turn, and the progress of the loop is recorded independently in each publisher, so two adjacent messages from different publishers may be consumed by the same client in the subscription group;
+ - `round_robin_per_group`: Clients in a shared subscription group will consume messages in turn, and the progress of the loop is recorded independently in each node, so two adjacent messages from different nodes may be consumed by the same client in the subscription group;
```

Comment:
this is wrong.
```suggestion
 - `round_robin_per_group`: Clients in a shared subscription group will consume messages in turn, and the progress of the loop is NOT recorded independently in each node, so two adjacent messages from different nodes are not consumed by the same client in the subscription group;
```

Reply:
In the code round_robin_per_group the progress of the round is stored inside the ETS table, so I think the statement that each node is recorded independently is correct.

Reply:
it's at least independent per node. otherwise the statement is exactly the same as `round_robin`.